src/Cake.Common/Net/HttpAliases.cs

```diff
@@ -197,7 +197,7 @@ public static void DownloadFile(this ICakeContext context, Uri address, FilePath
             // We track the last posted value since the event seems to fire many times for the same value.
             var percentComplete = 0;
 
-            using (var http = new HttpClient())
+            using (var http = new HttpClient(new HttpClientHandler { UseDefaultCredentials = settings.UseDefaultCredentials }))
```

Comment:
And what happens if both UseDefaultCredentials, and Username and Password are set?